### PR TITLE
Add application_choices to analytics.yml so their changes go to BigQuery

### PR DESCRIPTION
**Pull request by MTailor85**
Adding fields for the application_choices only as agreed to start with

## Link to Trello card

https://trello.com/c/wbvKXOZp/450-extract-events-from-apply-starting-with-the-fields-in-applicationchoices-table

### CODE REVIEW

**Comment by duncanjbrown** [comment]
Thanks @MTailor85!

I have 
- restored the `candidates:` section and the `- created_at` field inside
- removed the free text fields from the `application_choices` section

Testing locally, it looked like we didn't support sending Boolean (true/false) values from entity create/update events, so I've added support for that too

File: config/analytics.yml [hunk 1]
```diff
@@ -1,3 +1,31 @@
 shared:
   candidates:
     - created_at
+  application_choices:
```
Comment (misaka) [inline config/analytics.yml:4]: What if there's an error here? A typo in a fieldname, or a field gets removed from the schema? 

Are there tests?

**Review by duncanjbrown — APPROVED** [review]

File: config/analytics.yml (hunk 1 above)
Comment (duncanjbrown) [reply config/analytics.yml:4]: Good question. I'll try and add

Comment (duncanjbrown) [reply config/analytics.yml:4]: https://github.com/DFE-Digital/apply-for-teacher-training/pull/5195/commits/23747c95af2b257f213d00dedc7ef1710e1e0c48

**Review by cwrw — COMMENTED** [review]
@misaka @MTailor85 Hello, since @duncanjbrown is on holiday I'll try to address any comments on this PR. I pushed a commit https://github.com/DFE-Digital/apply-for-teacher-training/pull/5195/commits/2c9820a6ea00a3be2700b5fcebd90bf79ac3e914 to prevent us sending Big Query entity events unless we have the feature flag turned on, as this has caused some errors on other ENVs with the feature flag off: https://sentry.io/organizations/dfe-bat/issues/2515763493/events/c772071510e549039292c392369b13af/events/?project=1765973

Past that I think this PR is ready to be merged. Let me know if there are anything further comments. 

Regarding the feature flag, we are hoping to move away from it and use the presence of `BIG_QUERY_API_JSON_KEY` as an indicator instead, I'll open a followup PR after this is merged.

**Review by misaka — APPROVED** [review]
🚀

**Pull request merged**